code42.json

```diff
@@ -1924,6 +1924,463 @@
+                    "primary": false,
+                    "default": "",
+                    "order": 2
+                },
+                "subject": {
+                    "description": "Subject",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "order": 3
+                },
+                "assignee": {
+                    "description": "Assignee",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": true,
+                    "contains": [
+                        "email"
+                    ],
+                    "order": 4
+                },
+                "findings": {
+                    "description": "Findings (markdown)",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "order": 5
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.data.*.number",
+                    "data_type": "string",
+                    "contains": [
+                        "case number"
+                    ],
+                    "column_name": "Case number",
+                    "column_order": 0
+                },
+                {
+                    "data_path": "action_result.data.*.name",
+                    "data_type": "string",
+                    "column_name": "Case name",
+                    "column_order": 1
+                },
+                {
+                    "data_path": "action_result.data.*.description",
+                    "data_type": "string",
+                    "column_name": "Description",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.data.*.subjectUsername",
+                    "data_type": "string",
+                    "column_name": "Subject",
+                    "column_order": 3
+                },
+                {
+                    "data_path": "action_result.data.*.assigneeUsername",
+                    "data_type": "string",
+                    "column_name": "Assignee",
+                    "column_order": 4
+                },
+                {
+                    "data_path": "action_result.data.*.findings",
+                    "data_type": "string",
+                    "column_name": "Findings",
+                    "column_order": 5
+                }
+            ],
+            "render": {
+                "type": "table"
+            },
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "close case",
+            "identifier": "close_case",
+            "description": "Change the status of a Code42 case to 'CLOSED'",
+            "type": "investigate",
```

Comment:
I think the `type` here should be something else, probably `contain`.
I basically just make all the write operations `contain` and all the reads `investigate`